packages/cli/src/lib/flags/snapshotCommonFlags.ts

```diff
@@ -1,6 +1,11 @@
 import {flags} from '@oclif/command';
 import {Snapshot} from '../snapshot/snapshot';
 
+export enum PreviewLevel {
```

Comment:
```suggestion
export enum PreviewLevelValue {
```

IMO, it's a pitfall waiting for us to fall in to have two variables that changes by a simple capitalization*

*: except for classes and instances, but here, for e.g. it become fuzzy in the imports

Reply:
Could be `PreviewLevelOption` too if you prefer